packages/webapp/components/widgets/BestDiscussions.tsx

```diff
@@ -54,13 +54,11 @@ const ListItem = ({ post, onLinkClick }: PostProps): ReactElement => (
           style={{ width: `${post.featuredComments.length + 0.5}rem` }}
         >
           {post.featuredComments.map((comment, index) => (
-            <LazyImage
-              key={comment.id}
-              imgSrc={comment.author.image}
-              imgAlt={`${comment.author.name}'s profile picture`}
-              className="top-0 w-6 h-6 rounded-full"
-              absolute
-              style={{ left: `${index}rem` }}
+            <ProfilePicture
+              key={comment.author.username}
+              user={comment.author}
+              size="small"
+              className={`absolute left-${index}`}
```

Comment:
Not a good idea to use `left-${index}` as a class as the Tailwind JIT might remove this class when building the package. It looks for hardcoded classes and doesn't how to interpret dynamic ones. Let's keep the style prop instead.